atomsci/ddm/utils/datastore_functions.py

```diff
@@ -2087,13 +2094,13 @@ def copy_datasets_to_bucket(dataset_keys, from_bucket, to_bucket, ds_client=None
     if ds_client is None:
         ds_client = config_client()
 
-    if type(dataset_keys) == str:
+    if isinstance(dataset_keys, str):
         dataset_keys = [dataset_keys]
 
     # Copy each dataset
     for dataset_key in dataset_keys:
         try:
-            ds_meta = ds_client.ds_datasets.copy_dataset(bucket_name=from_bucket, dataset_key=dataset_key, to_bucket_name=to_bucket).result()
+            _ds_meta = ds_client.ds_datasets.copy_dataset(bucket_name=from_bucket, dataset_key=dataset_key, to_bucket_name=to_bucket).result()
```

Comment:
delete variable; needs test? I think it's ok to get something returned and just not assign it to anything?

Reply:
This will need to wait for a later release when the datastore is back up.